container/features/src/main/resources/features.xml

```diff
@@ -1077,4 +1077,12 @@
       <bundle>mvn:org.opennms.features.poller/org.opennms.features.poller.shell/${project.version}</bundle>
     </feature>
 
+    <feature name="opennms-telemetry-jti" description="OpenNMS :: Telemetry :: JTI" version="${project.version}">
+      <feature>opennms-collection-api</feature>
+      <feature version="${guavaVersion}">guava</feature>
+      <bundle>mvn:com.google.protobuf/protobuf-java/${protobufVersion}</bundle>
```

Comment:
Do we actually depend on protobuf?

Reply:
We do depend on protobuf in jti adapter.

Reply:
Duh, of course. I was confusing this with the protobuf messages used between OpenNMS and Minion.